### ISSUE TITLE
Chrome OS: A Linux or its own world?

### ISSUE THREAD

**Issue by dertuxmalwieder** [allinurl/goaccess#2080]
I noticed that Chrome OS, a Gentoo-based Linux distribution (cue terminology discussions here... ;-)), is listed as its own category. Is there a technical reason for that, especially as the PlayStation's operating system is listed as "BSD"?

**Comment by allinurl** [comment]
Good point. I think from the user's perspective, it would make more sense to show a that someone using Chrome OS visited their site as opposed to having it displayed as Gentoo. Like you said, maybe in that case it makes more sense to move PlayStation to "Others" as opposed to be under BSD. Thoughts?

**Comment by dertuxmalwieder** [comment]
Or maybe add a category for gaming consoles?

**Comment by allinurl** [comment]
That will work, probably the same as the browsers then.
